src/lib.rs

```diff
@@ -62,6 +62,8 @@
 //! impl private::Sealed for A {}
 //! ```
 
+use std::fmt::Display;
+
 use proc_macro::TokenStream;
```

Comment:
It would be better to reverse: use full path for `proc_macro::TokenStream` in the code and import short namings for `proc_macro2`, as the later appears in the code much more frequently.